18-references.Rmd

```diff
@@ -90,12 +90,10 @@ pkgs <- c(
   # JavaScript: 'DataTables'
 )
 
-# install packages if missing
-lapply(pkgs, function(pkg) {
-    if (system.file(package = pkg) != '') return()
-    install.packages(pkg, repos = "https://cloud.r-project.org/") 
-    }
-)
+# Warn to add new deps in DESCRIPTION
+pkg2 <- setdiff(pkgs, c("stats", "graphics", "grDevices", "utils", "datasets", "methods", "base"))
```

Comment:
I have an internal object `knitr:::.base.pkgs` that you may consider using here.

Reply:
Oh I look for such thing without success. Thanks ! 

```suggestion
pkg2 <- setdiff(pkgs, knitr:::.base.pkgs)
```